admin-manual/installation/installation.rst

```diff
@@ -444,7 +444,17 @@ Archivematica version 1.5.1 and higher support installation on CentOS/Redhat.
         [archivematica]
         name=archivematica
         baseurl=https://packages.archivematica.org/1.7.x/centos
-        gpgcheck=0
+        gpgcheck=1
+        baseurl=https://packages.archivematica.org/1.7.x/key.asc
```

Comment:
Here did you mean to use `gpgkey` instead?

Reply:
good catch!